pkg/vm/engine/disttae/db.go

```diff
@@ -15,9 +15,23 @@
 package disttae
 
 import (
+	"context"
+
 	"github.com/matrixorigin/matrixone/pkg/pb/plan"
+	"github.com/matrixorigin/matrixone/pkg/pb/timestamp"
+	"github.com/matrixorigin/matrixone/pkg/vm/engine"
 )
 
-func (db *DB) NewReader(databaseName, tableName string, expr *plan.Expr) Reader {
+func (db *DB) Update(ctx context.Context, databaseId, tableId uint64, ts timestamp.Timestamp) error {
```

Comment:
update what? only the timestamp?

Reply:
Update the version of a table to the specified timestamp。

Reply:
Do you need any other information than the timestamp?

Reply:
oh, when we get LogTailResponse,   we may need some interface to update DB's data

Reply:
Why this is a method of DB?   This *DB is shared by all sessions in CN?

Reply:
> Why this is a method of DB? This *DB is shared by all sessions in CN?
yes， cache is  multi-version, a cn has only one.

Reply:
> yes， cache is multi-version, a cn has only one.

yes， cache is multi-version, a cn has only one.